stories/forms/labels/fieldset-legend.story.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { FieldSetLegend } from 'src'
+
+storiesOf('FieldSetLegend', module)
+  .add('with default label', () => <FieldSetLegend name="nameOfInput" />)
+  .add('with custom label', () => (
+    <FieldSetLegend name="nameOfInput" label="Custom Label" />
+  ))
+  .add('with no label', () => (
+    <FieldSetLegend name="nameOfInput" label={false} />
+  ))
+  .add('with required true default indicator', () => (
```

Comment:
This makes it read like there is a required default indicator and I could see there being some confusion about nothing showing up since by default we do not show an indicator